test/core/end2end/end2end_test_corpus/seed_18.textproto

```diff
@@ -1,3 +1,6 @@
 suite: "CoreClientChannelTest"
 test: "SubchannelFilterCallInitFails"
 config: "Chttp2FakeSecurityFullstack"
+config_vars {
```

Comment:
Was this change intentional?

Reply:
yeah.... it's seed data for the fuzzer.... the generator picks up the current experiment list and puts them in **somewhere**... doesn't really matter where, more that the mutation engine gets to see that there's an experiment around.